### PR TITLE
last sync text color fixed (#574)

### PR DESCRIPTION
**Pull request by willbmartin**
Responding to issue #574 , added styling to make the "Synced at (time)" text black. First pull request, let me know how this looks!

### CODE REVIEW

**Comment by Natim** [comment]
Some of this were fixed with #639 can you rebase and remove the .black class from #639 to use your solution instead?

File: src/sidebar/editor.js [hunk 1]
```diff
@@ -170,6 +170,12 @@ function setAnimation( animateSyncIcon = true, syncingLayout, warning ) { // ani
   } else if (warning === false && footerButtons.classList.contains('warning')) {
     footerButtons.classList.remove('warning');
   }
+
+  if (syncSuccess === true && !footerButtons.classList.contains('syncSuccess')) {
+    footerButtons.classList.add('syncSuccess');
+  } else if (footerButtons.classList.contains('syncSuccess')) {
```
Comment (Natim) [inline src/sidebar/editor.js]: Can we change the if clause to have a else clause here?

File: src/sidebar/styles.css [hunk 2]
```diff
@@ -126,6 +126,10 @@ footer {
   color: black;
 }
 
+#footer-buttons.syncSuccess div:first-child {
```
Comment (Natim) [inline src/sidebar/styles.css]: I guess we can add the condition close in the previous CSS rule line 125.

**Comment by willbmartin** [comment]
@Natim, on it!

**Pull request merged**